### ISSUE TITLE
Recognizing separated words

### ISSUE THREAD

**Issue by agzam** [gennad/auto-highlight-symbol#3]
It's a question I unfortunately couldn't find answer for. It seems Spacemacs uses this package and when you highlight things like `foo/bar` it selects entire thing, whereas in most cases only a part needed. I'm wondering if it's possible to tell `ahs` to recognize slash as a separator? Thanks in advance.

**Comment by jcs090218** [comment]
I don't think this is how the package is intend to design in the first place. Hence I guess this wouldn't be a feature yet it does sounds possible.
